docs/src/app/components/pages/components/IconButton/README.md

```diff
@@ -0,0 +1,9 @@
+## Icon Button
+An [Icon Button](https://www.google.com/design/spec/components/buttons.html#) generates a button element around an icon. Also, focus styles will happen on tab but not on click. There are three ways to add an icon:
```

Comment:
What about this link https://www.google.com/design/spec/components/buttons.html#buttons-toggle-buttons instead?